app.py

```diff
@@ -39,19 +54,32 @@ def search():
     query = request.args.get('query', None)
     if query:
         print('query is %s' % query)
-        res = simple_match_search(ES, 'news', query)
-        list_res = res['hits']['hits']
-        for one in list_res:
-            # u6250082 Xuguang Song
-            sum_txt = body_summary(one['_source']['art'])
-            one['_source']['summary'] = ' '.join(sum_txt)
-        return jsonify(list_res)
+
+        in_result = rd.execute_command("cf.check", "test2", hash(query), fingerprint(query))
+        if in_result:
+            print('query is checked by cuckoofilter')
+            print('query is probably inside cuckoofilter')
```

Comment:
**[Required]** We shouldn't be including print statements in these endpoints. If we do need to log something, it's better to use a logger. Also we shouldn't be logging whether something is in or not in the cuckoo filter as it is redundant.

Reply:
I mean we can, but maybe not to this level, since this is captured by the redis module logs